lib/tests/ruby/ruby.cc

```diff
@@ -528,7 +528,7 @@ SCENARIO("custom facts written in Ruby") {
         }
     }
     GIVEN("a custom on_message block") {
-        log_capture capture(level::debug);
+        log_capture capture(level::warning);
```

Comment:
Hmm, file_readable emitting a debug message isn't great :( I'd probably change the capture.result() check to verify that the specific warning/debug messages in the test aren't present.